app/components/UI/FiatOnRampAggregator/Views/AmountToBuy.js

```diff
@@ -634,17 +645,10 @@ const AmountToBuy = () => {
       >
         <QuickAmounts
           onAmountPress={handleQuickAmountPress}
-          amounts={
-            currentFiatCurrency?.id === '/currencies/fiat/usd'
-              ? [
-                  { value: 100, label: '$100' },
-                  { value: 200, label: '$200' },
-                  { value: 300, label: '$300' },
-                  { value: 400, label: '$400' },
-                  // eslint-disable-next-line no-mixed-spaces-and-tabs
-                ]
-              : []
-          }
+          amounts={limits?.quickAmounts.map((limit) => ({
+            value: limit,
+            label: currentFiatCurrency?.denomSymbol + limit.toString(),
+          }))}
```

Comment:
Add a default empty array so `QuickAmounts` does not crash.
```suggestion
          amounts={limits?.quickAmounts.map((limit) => ({
            value: limit,
            label: currentFiatCurrency?.denomSymbol + limit.toString(),
          })) ?? []}
```

Reply:
All other options except for US, Pounds, and Euros will return an empty quickAmounts array. Should I still make that fallback?

Reply:
> Add a default empty array so `QuickAmounts` does not crash.

All other options except for US, Pounds, and Euros will return an empty quickAmounts array. Should I still make that fallback?

Reply:
Oh nice, then we don't 👌